core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -130,7 +130,7 @@ private[spark] class BlockManager(
     shuffleManager: ShuffleManager,
     val blockTransferService: BlockTransferService,
     securityManager: SecurityManager,
-    externalShuffleClient: Option[ExternalShuffleClient])
+    externalShuffleClient: Option[ExternalBlockStoreClient])
```

Comment:
rename the variable as well?

Reply:
Initially I tried this, but there are plenty of names bind with `Shuffle`, so in ae64538 I just change this param name and the member var `shuffleClient`.